showcase/src/app/demo-ui/pages/collapsible/demo-collapsible-page.component.html

```diff
@@ -21,44 +23,34 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		exampleTitle="SHOWCASE.DEMO.COLLAPSIBLE.CUSTOM_ICON"
 	>
 		<stark-collapsible
-			htmlId="secondCollapsible"
-			titleLabel="SHOWCASE.DEMO.COLLAPSIBLE.TITLE_1"
-			[(isExpanded)]="collapsed[1]"
-			[iconSpinningEnabled]="true"
-			icon="chevron-down"
```

Comment:
If we keep the "icon spinning" feature, I would say this example should not be removed but just change the icon and enable the icon spinning feature